### PR TITLE
fix(locale): pl building number and secondary address

### LINKED ISSUE #2103: Building number returns incorrect data for PL
### Pre-Checks

- [X] Follow our [Code of Conduct](https://github.com/faker-js/faker/blob/next/CODE_OF_CONDUCT.md).
- [X] Read the [Contributing Guidelines](https://github.com/faker-js/faker/blob/next/CONTRIBUTING.md).
- [X] Read the [docs](https://fakerjs.dev/guide).
- [X] Check that there isn't [already an issue](https://github.com/faker-js/faker/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a Faker issue and not related to a combination with another package.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/faker-js/faker/discussions) or join our [Discord Chat Server](https://chat.fakerjs.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.
- [ ] I am willing to provide a PR.

### Describe the bug

The implementation of `buildingNumber` for PL is incorrect because it can return i.e. 4/2 which is not a valid building number in Poland.
https://github.com/faker-js/faker/blob/next/src/locales/pl/location/building_number.ts

4/2 in Poland means:
- 4 is the home number
- 2 is the flat number

I think this applies also to most countries in Europe.

### Additional Context

We should remove `'#/#'` from `buildingNumber` patterns for PL.

### Environment Info

```shell
N/A
```

### Which module system do you use?

- [ ] CJS
- [X] ESM

### Used Package Manager

npm

### PR DESCRIPTION
**Pull request by matthewmayer**
fix #2103

Removes the X/Y pattern from building_number, and moves it to secondary_address instead. Also capitalizes a, b and c as it seems to be more common. Allow apartment numbers of between 1 and 3 digits for both the "m. #" and "/#" ways of writing the apartment number.

After these changes are implemented, some samples:
```
> fakerPL.helpers.multiple(fakerPL.location.buildingNumber, {count:100})
[
  '24',  '05B', '07',  '26C', '957', '839', '88C', '939',
  '23',  '35',  '65C', '577', '73B', '64',  '374', '18B',
  '62A', '78B', '59',  '54C', '997', '23B', '09C', '44A',
  '64B', '30C', '24B', '49',  '71',  '05A', '76A', '60A',
  '70A', '32B', '526', '89C', '78C', '33',  '94A', '00C',
  '14A', '75C', '13B', '57C', '177', '36B', '27C', '29',
  '90C', '35A', '26',  '74',  '21B', '770', '069', '670',
  '98A', '23',  '980', '68B', '37B', '814', '53C', '59B',
  '95B', '815', '29A', '58B', '54B', '03C', '06C', '89C',
  '359', '03C', '30',  '78',  '19C', '827', '76C', '86A',
  '83',  '700', '99A', '08A', '55A', '64A', '500', '62B',
  '82A', '70B', '29B', '23',  '999', '049', '99A', '685',
  '65C', '370', '75A', '892'
]
> fakerPL.helpers.multiple(fakerPL.location.streetAddress, {count:100})
[
  'skwer Lesiak 789',      'rondo Piątkowski 75C',   'szosa Walczak 48B',
  'rynek Buczyński 61A',   'rondo Staniszewski 73B', 'wyspa Trzciński 060',
  'park Jankowski 18A',    'ogród Grzybowski 59A',   'ul. Matuszak 38',
  'rynek Bąkowski 80B',    'rynek Kuchta 11C',       'szosa Milczarek 82',
  'bulw. Kowalczuk 37B',   'al. Sochacki 07A',       'os. Frączek 84',
  'os. Kowalczuk 972',     'os. Młynarczyk 12B',     'wyb. Lisiecki 23',
  'wyb. Korzeniowski 42A', 'rynek Chmielewski 87B',  'rynek Raczyński 76C',
  'ul. Michalczyk 80A',    'wyb. Borek 54C',         'ogród Nowiński 51C',
  'rondo Stachowicz 99B',  'wyspa Bochenek 49B',     'wyspa Pawelec 098',
  'ogród Wiącek 53C',      'ul. Socha 58A',          'wyspa Kozłowski 16A',
  'ogród Kruszewski 66A',  'wyb. Krysiak 12C',       'al. Głogowski 847',
  'al. Juszczak 35C',      'os. Frankowski 80C',     'pl. Partyka 33C',
  'bulw. Kopczyński 52A',  'ul. Wojtkowiak 625',     'rynek Trzeciak 98A',
  'droga Kałuża 47C',      'al. Musiał 07C',         'wyspa Zagórski 21C',
  'droga Cebula 13C',      'ogród Borkowski 34A',    'pl. Lewandowski 537',
  'pl. Kos 29C',           'bulw. Stachowicz 86',    'ul. Sosnowski 03A',
  'pl. Stachura 39B',      'szosa Mucha 20B',        'al. Kaczor 74',
  'rondo Konieczny 825',   'bulw. Kuliński 16B',     'ul. Maślanka 51',
  'szosa Buczkowski 44A',  'rondo Janicki 51B',      'szosa Piątek 71C',
  'bulw. Jankowiak 94C',   'droga Kania 59A',        'bulw. Stępień 76C',
  'ogród Sobczak 93B',     'os. Sawicki 649',        'ogród Kalisz 85B',
  'wyspa Szulc 83',        'wyb. Malec 165',         'al. Sawicki 964',
  'wyb. Gruszka 95',       'al. Dudziak 90B',        'ogród Szczęsny 99C',
  'os. Karwowski 88',      'rondo Kalisz 72C',       'wyspa Janas 963',
  'ul. Dąbkowski 384',     'wyb. Kurowski 30A',      'rondo Budziński 270',
  'pl. Bagiński 42B',      'ul. Sobieraj 91C',       'os. Kosiński 53A',
  'rondo Słowiński 935',   'szosa Krzemiński 048',   'park Pakuła 82B',
  'rynek Gruszka 27',      'wyspa Szczepaniak 24C',  'rynek Lipiński 24A',
  'park Polak 95',         'al. Jędrzejczyk 76B',    'szosa Borek 77C',
  'skwer Pietrzak 27',     'bulw. Markiewicz 614',   'ul. Bilski 207',
  'rynek Murawski 52C',    'wyb. Surma 42',          'os. Piórkowski 03C',
  'rynek Biernacki 89B',   'szosa Kruszewski 09C',   'ogród Książek 435',
  'ul. Czekaj 17A',        'os. Cybulski 20A',       'park Czajka 847',
  'szosa Kaczmarski 63A'
]
> fakerPL.helpers.multiple(()=>fakerPL.location.streetAddress(true), {count:100})
[
  'bulw. Fijałkowski 98 m. 045',
  'park Górka 657/956',
  'pl. Szyszka 49B/64',
  'rondo Rudnicki 257 m. 512',
  'pl. Tomala 71B/854',
  'ul. Chmielewski 09 m. 9',
  'droga Stępniak 42B/45',
  'skwer Krakowiak 98A/0',
  'rondo Wójcik 17C m. 485',
  'wyb. Wilczyński 10B m. 4',
  'wyb. Chojnacki 38C/89',
  'ogród Maćkowiak 69C m. 0',
  'bulw. Buczek 10A/20',
  'rondo Skowronek 906/8',
  'szosa Gołębiewski 91C/494',
  'skwer Chmiel 67C m. 26',
  'os. Owczarek 13C/99',
  'pl. Tomaszewski 00 m. 4',
  'ul. Szostak 253 m. 7',
  'szosa Kłos 61/4',
  'droga Kubica 00C/1',
  'ul. Pietrzak 15/382',
  'wyb. Pasternak 18C m. 5',
  'os. Maciejewski 18/33',
  'ogród Woźniak 07/09',
  'rondo Chrzanowski 331/24',
  'skwer Białek 99A m. 003',
  'ogród Trojanowski 42/27',
  'park Głogowski 03B m. 817',
  'bulw. Sikora 20C/246',
  'rondo Konieczny 857/839',
  'os. Bielecki 73 m. 0',
  'park Gawroński 02 m. 2',
  'os. Pietras 213/283',
  'wyspa Bielak 94A/08',
  'al. Borkowski 51C/212',
  'szosa Orzechowski 46/37',
  'wyspa Kulig 32A/73',
  'al. Kawa 61C/8',
  'wyspa Gutowski 60B m. 43',
  'pl. Hajduk 71 m. 8',
  'skwer Frąckowiak 92C m. 130',
  'al. Sławiński 668 m. 8',
  'ul. Mikulski 82C m. 642',
  'rondo Januszewski 14B/8',
  'bulw. Przybysz 15B m. 943',
  'droga Stachurski 970 m. 29',
  'szosa Kępa 24/944',
  'park Skalski 20A/8',
  'pl. Książek 66C m. 15',
  'droga Przybył 69A m. 0',
  'szosa Grzelak 36/46',
  'al. Gruszczyński 65C m. 6',
  'wyspa Kołodziejczyk 46A m. 7',
  'droga Raczyński 75A/0',
  'os. Koper 67B/1',
  'os. Leśniewski 13A m. 2',
  'park Ziętek 92/94',
  'ogród Dróżdż 844/4',
  'droga Lewicki 91A m. 90',
  'szosa Makowski 81B m. 156',
  'park Kaźmierczak 96A/930',
  'skwer Białek 99B m. 48',
  'os. Kubica 228/361',
  'os. Matuszewski 890/344',
  'droga Sadowski 57 m. 776',
  'szosa Kaczmarek 69B m. 94',
  'os. Jakubiak 319/2',
  'wyspa Kaczmarczyk 22B m. 1',
  'rondo Chrzanowski 41B m. 90',
  'os. Tracz 33C m. 4',
  'rondo Kawa 11/609',
  'park Sowiński 051 m. 457',
  'wyspa Banasiak 26A/824',
  'szosa Nowicki 93A/0',
  'droga Skrzypek 64 m. 022',
  'wyb. Górka 98/361',
  'os. Marzec 935/866',
  'droga Gajda 07 m. 5',
  'os. Kalisz 273 m. 84',
  'ogród Pawlik 08B/68',
  'os. Jurek 29A m. 00',
  'wyspa Paszkowski 915 m. 9',
  'park Miśkiewicz 11C m. 211',
  'al. Maćkowiak 00/035',
  'bulw. Adamowicz 16C m. 72',
  'park Polak 32A/55',
  'os. Szulc 98B/70',
  'pl. Antczak 37 m. 6',
  'park Zając 81A m. 85',
  'ul. Pietrzak 52C/142',
  'al. Janas 147 m. 8',
  'droga Ostrowski 22C m. 11',
  'szosa Masłowski 38C m. 0',
  'os. Partyka 09/7',
  'szosa Kruk 922/8',
  'al. Kuliński 96/77',
  'ul. Furman 51B m. 852',
  'wyb. Filipek 53C/3',
  'szosa Kamiński 41 m. 12'
]
```

### CODE REVIEW

File: src/locales/pl/location/secondary_address.ts [hunk 1]
```diff
@@ -1 +1 @@
-export default ['m. ###'];
+export default [' m. #', ' m. ##', ' m. ###', '/#', '/##', '/###'];
```
Comment (ST-DDT) [inline src/locales/pl/location/secondary_address.ts:1]: **EDIT**

IMO we should think about what the secondary address part is actually used for.
E.g. Is it supposed to be a suffix or is it something standalone? (In Germany, it is usually standalone (separate line), our patterns indicate it is a suffix though. 🤔

Comment (matthewmayer) [reply src/locales/pl/location/secondary_address.ts:1]: its definitely locale dependent, whether it is a suffix or a prefix etc depends on the streetAddress.full pattern. i think whether it is a newline would require something like i proposed in #2011

File: src/locales/pl/location/building_number.ts
Comment (rybaczewa) [inline src/locales/pl/location/building_number.ts:1]: Removal of `#/#` pattern was based on issue, where the author was unaware, that it is a valid building number pattern. https://github.com/faker-js/faker/issues/2103#issuecomment-1535875003

Example: https://goo.gl/maps/bqHN1W3PpEJiCZer9

**Comment by matthewmayer** [comment]
Closing as there is no consensus to change this.

**Pull request closed without merging**